configs/basecall.config

```diff
@@ -0,0 +1,38 @@
+/************************************************
+| CONFIGURATION FILE FOR NAO BASECALL WORKFLOW |
+************************************************/
+
+params {
+    mode = "basecall"
+    debug = true
+
+    // Directories
+    base_dir = "s3://nao-mgs-simon/NAO-ONT-20240905-DCS_RNA2_Test"
+
+    // Run parameters
+    // TODO: Automate getting run name from base_dir
+    nanopore_run = "NAO-ONT-20240905-DCS_RNA2_Test"
```

Comment:
If this value is always equivalent to the basename of base_dir, you can do this with Groovy: https://www.nextflow.io/docs/latest/working-with-files.html#getting-file-attributes